web/src/components/Incidents/IncidentsPage.jsx

```diff
@@ -113,17 +111,23 @@ const IncidentsPage = () => {
   );
   React.useEffect(() => {
     const hasUrlParams = Object.keys(urlParams).length > 0;
-
     if (hasUrlParams) {
       // If URL parameters exist, update incidentsActiveFilters based on them
       dispatch(
         setIncidentsActiveFilters({
           incidentsActiveFilters: {
-            ...incidentsInitialState,
-            ...urlParams,
+            days: urlParams.days ? urlParams.days : ['15 days'],
```

Comment:
Can you please change to 7 days, as it happens when you land with no params at all?

```suggestion
            days: urlParams.days ? urlParams.days : ['7 days'],
```